src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -59,6 +60,10 @@
 
   private final String alreadyConvertedPrefix;
 
+  // Used to store destructuring assignments like "const {a, b} = abModule;".
+  // Later on we use this map to rewrite full module imports.
+  private final Map<String, Node> destructuringAssignments = new HashMap<>();
```

Comment:
Can this be more specific than Node, I think you only use `ImportSpec` as values

Reply:
I'm not sure how to specify `ImportSpec` types here. `Node` does not take generic types.
I'm assuming that you meant to explicitly call out `ImportSpec` in the comments of this Map.

Reply:
oh, forgot closure AST is untyped :/